src/redux/reducers/covid.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable no-case-declarations */
+import _ from 'lodash';
+import {
+  FETCH_CONTINENT,
+  FETCH_CONTINENT_FAILURE,
+  FETCH_COUNTRY,
+} from '../actions/action';
+
+const initialState = [];
+
+const vacineReducer = (state = initialState, { type, payload }) => {
+  switch (type) {
+    case FETCH_CONTINENT:
+      return payload;
+    case FETCH_CONTINENT_FAILURE:
+      return state;
+    case FETCH_COUNTRY:
+      const newState = _.filter(
+        state,
+        (vaccine) => vaccine.All.country === payload,
+      );
+      return newState;
+    default:
+      return state;
+  }
+};
+
+export default vacineReducer;
```

Comment:
- [ ] You have to connect the Redux store, using the `connect()`

Reply:
Hi @Emmyn5600,

Please elaborate more on this. Do you mean I cannot connect Redux with react as stated in this [document](https://react-redux.js.org/api/hooks)?